REVIEW_GUIDELINES.md

```diff
@@ -76,6 +76,45 @@ image:
 * The use of the `default` function should be avoided if possible in favor of defaults in `values.yaml`.
 * It is usually best to not specify defaults for resources and to just provide sensible values that are commented out as a recommendation, especially when resources are rather high. This makes it easier to test charts on small clusters or Minikube. Setting resources should generally be a conscious choice of the user.
 
+## Persistence
+
+* Persistence should be turned off by default
+* PVCs should support specifying an existing claim
+* Use multi-attach PVCs instead of several small pvcs
+
+
+```yaml
+volumeMounts:
+  - mountPath: /opt/sonarqube/conf
+    name: sonarqube
+    subPath: conf
+  - mountPath: /opt/sonarqube/data
+    name: sonarqube
+    subPath: data
+  - mountPath: /opt/sonarqube/extensions
+    name: sonarqube
+    subPath: extensions
```

Comment:
Maybe don't make it SonarQube-specific. A simpler example should suffice. I'd also remove the `subPath` from the example. It's optional and could be misleading.

Reply:
Yep. Is there a better place to put the subPath example? I agree it might be confusing here, but I would like to document it somewhere for people to use.

Reply:
Don't know where to put it. I think it is not specific to charts and we don't have to repeat Kubernetes docs here. I'd remove it.

Reply:
Makes sense. Done.